docker/docker_tools.py

```diff
@@ -248,12 +286,39 @@ def build(
             print(f"Unable to run git: {e} Installing...")
             xqt("sudo apt-get install -y --no-install-recommends git")
 
+        # If Clone-all flag is set override other Clone Flags
+        if clone_all != "RunestoneInteractive":
+            # Print Warning and provide countdown to abort script
+            click.secho(
+                "Warning: Clone-all flag was initalized and will override any other clone flag specifed!",
+                fg="red",
+            )
+            from time import sleep
+
+            # Wait 20 Seconds prior to proceeding with script
```

Comment:
I'd omit this. A warning in red is enough.

Reply:
Commit 4bccb7c024d88799ceed7007694838722406baf1 removes delay.